ts/features/itwallet/screens/discovery/ItwDiscoveryInfoScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { ContentWrapper } from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n";
+import { useHeaderSecondLevel } from "../../../../hooks/useHeaderSecondLevel";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import ItwMarkdown from "../../components/ItwMarkdown";
+import { RNavScreenWithLargeHeader } from "../../../../components/ui/RNavScreenWithLargeHeader";
+import { FooterStackButton } from "../../components/FooterStackButton";
+
+/**
+ * This is the screen that shows the information about the discovery process
+ * about the activation of the DIW. It uses a markdown component to render
+ * the content of the screen. The screen is wrapped in a GradientScrollView
+ * with a primary and secondary action.
+ */
+const ItwDiscoveryInfoScreen = () => {
+  useHeaderSecondLevel({
+    title: I18n.t("features.itWallet.discovery.title"),
+    contextualHelp: emptyContextualHelp,
+    supportRequest: true
+  });
+
+  return (
+    <RNavScreenWithLargeHeader
+      title={{ label: I18n.t("features.itWallet.discovery.title") }}
+      fixedBottomSlot={
+        <FooterStackButton
+          primaryActionProps={{
+            label: I18n.t("global.buttons.continue"),
+            accessibilityLabel: I18n.t("global.buttons.continue"),
+            onPress: () => undefined
+          }}
+          secondaryActionProps={{
+            label: I18n.t("global.buttons.cancel"),
+            accessibilityLabel: I18n.t("global.buttons.cancel"),
+            onPress: () => undefined
+          }}
+        />
+      }
+    >
+      <ContentWrapper>
+        {/* Info activation */}
+        <ItwMarkdown content={I18n.t("features.itWallet.discovery.content")} />
+      </ContentWrapper>
+    </RNavScreenWithLargeHeader>
+  );
+};
+
+export default ItwDiscoveryInfoScreen;
```

Comment:
```suggestion
export { ItwDiscoveryInfoScreen };
```

Even though this component is used only within the navigator component, using a named export will help us to have consistency. What do you think?

Reply:
Yep good point thank you

Reply:
Solved [here](https://github.com/pagopa/io-app/pull/5696/commits/9abd9c5fa423294ca87f32940834424ea7351660)